pages/Messages/Overview.md

```diff
@@ -175,83 +99,22 @@ A visual reply to a post.
 | dsnpData field | description | type | bloom |
 | ------------- |------------- | ---- | --- |
 | inReplyTo | ID of the message the reaction references |  bytes32 | YES
```

Comment:
I think these inReplyTos are incorrect now. They should be string style identifiers: https://spec.projectliberty.io/Messages/Identifiers